.github/workflows/master-snapshot-release.yml

```diff
@@ -29,9 +29,80 @@ jobs:
         run: ./mvnw ${MAVEN_ARGS} -B test --file pom.xml
       - name: Run integration tests
         run: ./mvnw ${MAVEN_ARGS} -B package --file pom.xml
-  release-snapshot:
+  spring-boot-integration-test:
     runs-on: ubuntu-latest
     needs: test
+    strategy:
```

Comment:
Would not run this on snapshot release, it is already run on PR at this point.

Reply:
I understood you asked for adding this test as part of the release job. 
Anyway, deleted and PR updated.